snews_cs/snews_coinc.py

```diff
@@ -416,14 +411,13 @@ def hype_mode_publish(self):
         for _sub_group, new_message_count in (new_count - self.coinc_data.old_count).iteritems():
             if new_message_count == 0:
                 continue
-
+            if self.coinc_data.msg_state == 'RETRACTION':
```

Comment:
Again, haven't tested yet, Just by looking at the code, should this be `self.coinc_data.msg_state` or `self.msg_state` isn't `self.coinc_data` a dataframe?

Reply:
Nope, looking at the whole code, I just noticed that there are two classes, and this usage is actually correct.